HelpSource/Reference/ServerPluginAPI.schelp

```diff
@@ -0,0 +1,189 @@
+title:: Server Plugin API
```

Comment:
@snappizz the advantage of having this documentation in the doxygen documentation:

* one can easily know what's documented and what not
* no spreading of information
* available in the place where you need it when writing c++ (i doubt anyone will write c++ code in scide)

fwiw, this is getting on a side track: neither unroll macros, nor the inline functions are actually part of the actual plugin interface, which basically boils down to the stuff in https://github.com/supercollider/supercollider/blob/master/include/plugin_interface/SC_InterfaceTable.h. the rest is basically helper functionality

--

@bagong it is always better to try to do things right in the first place. that's what code reviews (or documentation reviews) are for.